operatorcourier/validate.py

```diff
@@ -79,9 +111,9 @@ def _bundle_validation(self, bundle, repository=None):
         if validationDict[self.pkgsKey] and repository is not None:
             packageName = bundleData['packages'][0]['packageName']
             if repository != packageName:
-                self._log_error('The packageName (%s) in bundle does not match '
-                                'repository name (%s) provided as command line argument.',
-                                packageName, repository)
+                self._log_error(f'The packageName ({packageName}) in bundle does not '
```

Comment:
nit: Is this change required? If all logs are switching to this format, it's fine to me

Reply:
@SamiSousa Because I added a 2nd param to `_log_error(self, message, message_kind='', *args, **kwargs)`. With the original log format, `packageName` will be wrongly passed as **`message_kind`**, which will cause issue.